src/s2/s1angle_test.cc

```diff
@@ -71,16 +70,16 @@ TEST(S1Angle, E6E7RepresentationsUnsigned) {
   // Check that unsigned E6/E7 representations work as expected.
   EXPECT_DOUBLE_EQ(
       S1Angle::Degrees(60).radians(),
-      S1Angle::UnsignedE6(static_cast<uint32>(60000000)).radians());
+      S1Angle::UnsignedE6(static_cast<std::uint32_t>(60000000)).radians());
```

Comment:
Let's do the same `using std::uint32_t;` thing in the .cc files that #92 does with `std::string`.